### DISCUSSION TITLE
mysql_native_password not necessary

### DISCUSSION THREAD

**Discussion by tessus** [dani-garcia/vaultwarden#4247] [Q&A]
The wiki page [Using the MariaDB (MySQL) Backend](https://github.com/dani-garcia/vaultwarden/wiki/Using-the-MariaDB-(MySQL)-Backend) states the following:

```sql
-- Use this on MySQLv8 installations
CREATE USER 'vaultwarden'@'localhost' IDENTIFIED WITH mysql_native_password BY 'yourpassword';
```

`mysql_native_password` has been deprecated for a while and one gets a lot of warning messages in the mysql log.

May I ask why the wiki states that `mysql_native_password` should be used? I've changed it days ago to `caching_sha2_password` and it works just fine.

**Comment by stefan0xC** [comment]
Because the wiki doesn't update itself? Feel free to change it, especially if it's deprecated. (I'm not using the MariaDB backend, so I rarely check related pages myself.)

**Comment by BlackDex** [comment]
The reason was (or maybe still is), that the MariaDB version supported during the time that was written didn't supported the new way, and caused it to error out.

I think i have enabled that password option in the Alpine builds for a long time already, but the older base images just didn't supported it.
So, please test if the new `:latest` tagged images, both Debian and Alpine, work with the new password option, if not, please let us know, and leave the wiki as-is, else, feel free to adjust it :).

**Reply by tessus** [reply]
> So, please test if the new :latest tagged images, both Debian and Alpine, work with the new password option, if not, please let us know, and leave the wiki as-is, else, feel free to adjust it :).

I currently don't have any docker envs running anywhere. I can look into it, but I don't know when I will get to it.
